lib/WebDriverKeys.php

```diff
@@ -90,11 +90,13 @@ class WebDriverKeys
     const COMMAND = self::META;
 
     /**
-     * Encode input of `sendKeys()`.
+     * Encode input of `sendKeys()` to appropriate format according to protocol.
+     *
      * @param string|array|int|float $keys
-     * @return array
+     * @param bool $isW3cCompliant
+     * @return array|string
      */
-    public static function encode($keys)
+    public static function encode($keys, $isW3cCompliant = false)
```

Comment:
For future, we should really rename this to normalise since that it what is happening. No values are encoded.